package.json

```
@@ -9,10 +11,11 @@
     "mdlint": "markdownlint-cli2 '**/*.md' '!node_modules' '!meetings/201*/*.md' '!meetings/202[0-2]*/*.md' '!meetings/2023-0[1-3]/*.md'",
     "mdlint:fix": "markdownlint-cli2-fix '**/*.md' '!node_modules' '!meetings/201*/*.md' '!meetings/202[0-2]*/*.md' '!meetings/2023-0[1-3]/*.md'",
```

The `markdownlint-cli2-fix` [is deprecated](https://github.com/DavidAnson/markdownlint-cli2#command-line). Use `--fix` option instead.
```suggestion
    "mdlint:fix": "markdownlint-cli2 --fix '**/*.md' '!node_modules' '!meetings/201*/*.md' '!meetings/202[0-2]*/*.md' '!meetings/2023-0[1-3]/*.md'",
```

---

that's new -- not available in this version.  we can upgrade but let's do that in a separate PR